Automattic/wp-calypso
# Apply introductory offer on on the flow that starts from move page

Related to https://github.com/Automattic/dotcom-forge/issues/8280

## Proposed Changes

* If the user comes to the checkout page with the flow beginning from the `move-lp` (https://wordpress.com/move) page, we mark the site with the introductory sticker to apply the introductory offer discount on the Creator plan.

I haven't written tests for it yet to get feedback first and unblock (check the thread link on the issue) other tasks.

![image](https://github.com/user-attachments/assets/6a3bcec5-69f3-431d-b2d2-8111a3401967)

## Why are these changes being made?


* To make our introductory offer on migration more visible and actually apply the discount.

## Testing Instructions



- Apply the diff D155293-code
- Make sure you are sandboxed
- Go to https://wordpress.com/move
- Make sure you see the discount on the Creator plan
- Click on the "Get Creator" button
- Complete the checkout flow and make sure you saw the discount applied

Try the steps above as both logged-in and logged-out users. If "https://wordpress.com/start/business/?ref=move-lp" shows error, replace https://wordpress.com with calypso.localhost:3000

## Pre-merge Checklist
